### PR TITLE
Apple Silicon Builds (aarch64)

### PR DESCRIPTION
**Pull request by yougotwill**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -18,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, macos-12, ubuntu-20.04]
```
Comment (Bilb) [inline .github/workflows/pull-request.yml:21]: macos-13-xlarge is something we pay for, please remove this for pull-request

File: .github/workflows/build-binaries.yml [hunk 2]
```diff
@@ -22,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, macos-12, ubuntu-20.04]
```
Comment (Bilb) [inline .github/workflows/build-binaries.yml:25]: macos-13-xlarge is something we pay for, please remove this for build binaries
and remove the change below too, as undeeded

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, macos-12, ubuntu-20.04]
+        os: [windows-2022, ubuntu-20.04, macos-12, macos-13-xlarge]
```
Comment (Bilb) [inline .github/workflows/release.yml:15]: this one can be kept

Comment (Bilb) [reply .github/workflows/release.yml:15]: did you make sure that the binaries were not overriding each others once uploaded?

**Review by Bilb — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (yougotwill) [reply .github/workflows/release.yml:15]: Confirmed the binaries are correct for each platform.

**Review by Bilb — APPROVED** [review]

**Review by yougotwill — COMMENTED** [review]
LGTM 🚢

**Pull request merged**